_posts/2024-09-05-FEX-2409.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: FEX 2409 Tagged
+author: FEX-Emu Maintainers
+---
+
+FEX-2409 is now released... with a huge performance boost.
+
+[GEEKBENCH RESULTS HERE]
+
+## I'm tired, carry me.
+
+Little differences between x86 and Arm can cause big performance penalties for
+emulators. None more so than flags.
+
+What are those?
+
+Flags are bits representing the processor state. Both x86 and Arm have flags.
```

Comment:
You're elaborating below, but if we're giving this introduction then it's probably useful to directly name a couple of examples. "Processor state" could mean anything otherwise. E.g.:

```suggestion
Flags are bits representing the processor state, such as comparison results or overflow indicators. Both x86 and Arm have flags.
```